schemas/2016-04-30-preview/Microsoft.Compute.json

```diff
@@ -0,0 +1,4781 @@
+    },
+    "VirtualMachineScaleSetExtensionProfile": {
+      "type": "object",
+      "properties": {
+        "extensions": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/VirtualMachineScaleSetExtension"
+              }
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The virtual machine scale set child extension resources."
+        }
+      },
+      "description": "Describes a virtual machine scale set extension profile."
+    },
+    "VirtualMachineScaleSetExtensionProperties": {
+      "type": "object",
+      "properties": {
+        "publisher": {
+          "type": "string",
+          "description": "The name of the extension handler publisher."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the extension handler."
+        },
+        "typeHandlerVersion": {
+          "type": "string",
+          "description": "The type version of the extension handler."
+        },
+        "autoUpgradeMinorVersion": {
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Whether the extension handler should be automatically upgraded across minor versions."
+        },
+        "settings": {
+          "oneOf": [
+            {
+              "type": "object"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Json formatted public settings for the extension."
+        },
+        "protectedSettings": {
+          "oneOf": [
+            {
+              "type": "object"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Json formatted protected settings for the extension."
+        }
+      },
+      "description": "Describes the properties of a Virtual Machine Scale Set Extension."
+    },
+    "VirtualMachineScaleSetIPConfiguration": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "description": "The IP configuration name."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/VirtualMachineScaleSetIPConfigurationProperties"
+    
```

Comment:
"upgradePolicy" and "virtualMachineProfile" are required properties in the previous. Can we confirm on the change?